src/main/java/productDependency/CreateCustomerPage.java

```diff
@@ -0,0 +1,172 @@
+package productDependency;
+
+import baseClassPackage.BasePage;
+import categoryProducts.AddProductPage;
+import categoryProducts.AddProductWithAssetPage;
+import companyHierarchy.ViewAssignedInfoPage;
+import productDependency.CreateCustomerPage;
```

Comment:
@mayankchhajer remove the unused imports.

Reply:
Done